v2/tools/generator/internal/jsonast/swagger_type_extractor.go

```diff
@@ -953,3 +955,39 @@ func makeSchemaFromSimpleSchemaParam(param schemaAndValidations) *spec.Schema {
 
 	return schema
 }
+
+var (
+	resourceGroupScopeRegex = regexp.MustCompile(`(?i)^/subscriptions/[^/]+/resourcegroups/[^/]+/.*`)
+	locationScopeRegex      = regexp.MustCompile(`(?i)^/subscriptions/[^/]+/.*`)
+	resourceScopePrefixes   = []string{
```

Comment:
Shouldn't this be something like 
```suggestion
	extensionScopePrefixes   = []string{
```

Reply:
Also somewhat minor but in this instance you could probably get away with a single regex like:
`(?i)^/({scope}|{resourceUri})/.*` (might need to escape the `{`'s though)

Main advantage of this would just be to simplify the scope check at least for now.

Reply:
I considered that but figured to take the more explicit approach.